src/tests/Common/CLRTest.Jit.targets

```diff
@@ -187,6 +192,108 @@ IF NOT DEFINED DoLink (
     </PropertyGroup>
   </Target>
 
+  <Target Name="GetDisasmCheckData"
+          Returns="$(HasDisasmCheck);@(DisasmCheckFiles)">
+    <ItemGroup>
+      <DisasmCheckFiles Include="%(Compile.Identity)" Condition="'%(Compile.HasDisasmCheck)' == 'true'" />
+    </ItemGroup>
+    <PropertyGroup>
+      <HasDisasmCheck>false</HasDisasmCheck>
+      <HasDisasmCheck Condition="@(DisasmCheckFiles->Count()) &gt; 0 And '$(CLRTestKind)' == 'BuildAndRun' And '$(TargetOS)' == 'windows' And '$(TargetArchitecture)' == 'x64'">true</HasDisasmCheck>
+    </PropertyGroup>
+  </Target>
+
+  <Target Name="PropagateHasDisasmCheckToCopy"
+          BeforeTargets="AssignTargetPaths">
+    <ItemGroup>
+      <Compile Update="@(Compile)"
+               Condition="'%(Compile.HasDisasmCheck)' == 'true'"
+               CopyToOutputDirectory="PreserveNewest" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="GetDisasmCheckBashScript"
+          Returns="$(HasBashDisasmCheck);$(BashDisasmOutputFile);$(BashDisasmCheckPostCommands)"
+          DependsOnTargets="GetDisasmCheckData">
+    <PropertyGroup>
+      <HasBashDisasmCheck>$(HasDisasmCheck)</HasBashDisasmCheck>
+
+      <BashDisasmOutputFile>/dev/null</BashDisasmOutputFile>
+      <BashDisasmOutputFile Condition="'$(HasBashDisasmCheck)' == 'true'">%scriptPath%/__jit_disasm.out</BashDisasmOutputFile>
+
+      <BashDisasmListOutputFile>/dev/null</BashDisasmListOutputFile>
+      <BashDisasmListOutputFile Condition="'$(HasBashDisasmCheck)' == 'true'">%scriptPath%/__jit_disasm_list.out</BashDisasmListOutputFile>
+
+      <BashDisasmCheckPostCommands></BashDisasmCheckPostCommands>
+      <BashDisasmCheckPostCommands Condition="'$(HasBashDisasmCheck)' == 'true'"><![CDATA[
+if [[ -n "$RunningDisasmChecks" ]]
+then
+@(DisasmCheckFiles -> '  $(ArtifactsDir)bin/SuperFileCheck/$(TargetArchitecture)/$(Configuration)/$(OutputRid)/SuperFileCheck --csharp "%(Identity)" --allow-unused-prefixes --check-prefixes=CHECK,$(TargetArchitecture.ToUpperInvariant()) --input-file "$(BashDisasmOutputFile)"
+  ERRORLEVEL=$?
+  if [[ $ERRORLEVEL -ne 0 ]]
+  then
+    echo EXECUTION OF FILECHECK - FAILED $ERRORLEVEL
+    exit 1
+  fi', '%0a')
+fi]]>)
+      </BashDisasmCheckPostCommands>
```

Comment:
I think it would be more readable to put the "PostCommands" after the "PreCommands" in this file.  (bash and batch)

Reply:
Yes, I agree.